dmcontent/content_loader.py

```diff
@@ -131,6 +131,7 @@ def create(cls, section):
             return ContentSection(
                 slug=section['slug'],
                 name=section['name'],
+                prefill=section.get('prefill'),
```

Comment:
I thought about that but I thought a) I would follow what `editable` does and b) someone someday might want to be able to discriminate between "not set" and "explicitly False"...